.idea/workspace.xml

```diff
@@ -0,0 +1,1296 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is ignored but also commited. That shouldn't be the case (it should be ignored)

Reply:
hmm - will delete that locally - let me take another look.